hatasmota/status_sensor.py

```diff
@@ -74,6 +77,7 @@
     SENSOR_STATUS_LINK_COUNT: "WiFi Connect Count",
     SENSOR_STATUS_MQTT_COUNT: "MQTT Connect Count",
     SENSOR_STATUS_RESTART_REASON: "Restart Reason",
+    SENSOR_BATTERY: "Battery Level",
```

Comment:
We don't need a name for this, Home Assistant automatically sets the name of battery sensors
```suggestion
```

Reply:
Never mind this comment, the Tasmota integration has not yet opted in to this feature